### PR TITLE
Add Dragon Ball Z: Extreme Butōden

### PR DESCRIPTION
**Pull request by MojoJojoDojo**
Adds Dragon Ball Z: Extreme Butōden.

*Automatically generated by PR builder*

### CODE REVIEW

File: games/dragon-ball-z-extreme-butden/game.dat [hunk 1]
```diff
@@ -0,0 +1,19 @@
+title = "Dragon Ball Z: Extreme Butōden"
+description = "A 2D Fighting Game based on the popular anime Dragon Ball and released by Bandai Namco."
```
Comment (Leo626) [inline games/dragon-ball-z-extreme-butden/game.dat]: This sounds a bit better.
```
Dragon Ball Z: Extreme Butōden is an 2D fighting game developed by Bandai Namco and is based on the popular anime, Dragon Ball.
```

File: games/dragon-ball-z-extreme-butden/game.dat [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+[[releases]]
+title = "000400000016CD00"
+region = "usa"
+release_date = "2015-10-20"
+
+[[testcases]]
+title = "000400000016CD00"
+compatibility = "0"
+date = "2017-11-09"
+version = "HEAD-89b9a4b"
```
Comment (Leo626) [inline games/dragon-ball-z-extreme-butden/game.dat]: Testcases should be done with Nightly.

File: games/dragon-ball-z-extreme-butden/game.dat [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+cpu = "Core 2 Duo E6750"
```
Comment (Leo626) [inline games/dragon-ball-z-extreme-butden/game.dat]: Should add `Intel` to the beginning.

File: games/dragon-ball-z-extreme-butden/game.dat [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+gpu = "Geforce GT 630"
```
Comment (Leo626) [inline games/dragon-ball-z-extreme-butden/game.dat]: Should add `Nvidia` to the beginning.

File: games/dragon-ball-z-extreme-butden/game.dat [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+os = "Windows 10"
```
Comment (Leo626) [inline games/dragon-ball-z-extreme-butden/game.dat]: Should mention Windows 10 version. Run `cmd` to find it.

**Comment by Leo626** [comment]
* 3 screenshots minimum.
* Nitpick, the boxart has a watermark at the bottom right. A more suitable one would be this [one](https://gamefaqs.akamaized.net/box/9/5/2/572952_front.jpg) resized accordingly.
* The wiki should follow the example in the [Contributing](https://github.com/citra-emu/citra-games-wiki/blob/master/CONTRIBUTING.md#wiki) guide.

**Comment by MojoJojoDojo** [comment]
Changed what you requeted by i dont know how to change/add the screenshots.

**Comment by Leo626** [comment]
To keep it simple, go to the [directory](https://github.com/MojoJojoDojo/citra-games-wiki/tree/pr-builder-dragon-ball-z-extreme-butden-91/games/dragon-ball-z-extreme-butden/) in your repo and either drag and drop the files or use the Upload files button. You'll want to change the boxart to the one recommended.

**Comment by MojoJojoDojo** [comment]
done

**Comment by Leo626** [comment]
- The second screenshot has a weird icon at the bottom left corner.
- The last screenshot has the mouse cursor. Also, can you use a different scene/fighter? It looks too similar to the second one.
- You can use ShareX's Capture Region feature to take screenshots.

**Comment by MojoJojoDojo** [comment]
I changed the pictures agein but i dont know how to rename the folder/directory.
sorry for the trouble.

**Pull request closed without merging**